bench/src/bulk.rs

```diff
@@ -274,6 +274,7 @@ fn transport_config(opt: &Opt) -> quinn::TransportConfig {
     config
         .max_concurrent_uni_streams(opt.max_streams as u64)
         .unwrap();
+    config.congestion_controller_factory(Arc::new(quinn_proto::congestion::CubicConfig::default()));
```

Comment:
I doubt this makes a difference in the benchmark without latency injection. Maybe if socket buffers are set rather low, where New Reno will back off with a sawtooth profile once the socket buffer size had been hit. But with a more realistic setting this doesn't happen.

Probably needs some kind of loss or latency injection to really show the benefit.

Reply:
Right that part was just so I had an easy test that it was working. I can revert this.